pandas/core/common.py

```diff
@@ -1764,6 +1764,35 @@ def interpolate_2d(values, method='pad', axis=0, limit=None, fill_value=None, dt
     return values
 
 
+def interpolate_nd(values, method='pad', axis=0, limit=None, fill_value=None, dtype=None):
```

Comment:
if you are going to do this, ok, but I think a reorg of all of these little functions is in order. If before this PR (e.g. send another), put all of the interp methods that are in ``core/common.py`` into ``core/missing.py`` would be helpful